File: LeetCode Solutions/Array and String/Max Consecutive Ones/Max Consecutive Ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int countOfOne = 0, maximumCountOfOne = 0;
        for (int digit : nums){
            if (digit == 1){
                countOfOne++;
                maximumCountOfOne = max(maximumCountOfOne, countOfOne);
            } else{
                countOfOne = 0;
            }
        }
        return maximumCountOfOne;
    }
};

File: LeetCode Solutions/LeetCoding Challenge/September 2020 LeetCoding Challenge/Max Consecutive Ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        result = count = 0
        nums.append(0)
        for i in nums:
            if i == 1:
                count += 1
            else:
                result = max(result, count)
                count = 0
        return result